infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultLogReplicationConfigAdapter.java

```diff
@@ -1,12 +1,28 @@
 package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
 
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.TxnContext;
+
 /**
  * Default testing implementation of a Log Replication Config Provider
  */
 public class DefaultLogReplicationConfigAdapter implements ILogReplicationVersionAdapter {
 
+    private static final String latestVer = "LATEST-VERSION";
+
+    public DefaultLogReplicationConfigAdapter(CorfuRuntime runtime) {
```

Comment:
since we are passing the txn context, is the runtime required?

Reply:
yes the runtime is needed to instantiate the corfuStore instance which will be used to openTable()s initially. If we do this we can avoid attempting to open tables on access.